packages/@react-aria/selection/src/useSelectableItem.ts

```diff
@@ -66,10 +76,18 @@ export function useSelectableItem(options: SelectableItemOptions): SelectableIte
     shouldSelectOnPressUp,
     isVirtualized,
     shouldUseVirtualFocus,
-    focus
+    focus,
+    isDisabled,
+    onAction
   } = options;
 
-  let onSelect = (e: PressEvent | PointerEvent) => manager.select(key, e);
+  let onSelect = (e: PressEvent | LongPressEvent | PointerEvent) => {
+    if (e.pointerType === 'keyboard' && isNonContiguousSelectionModifier(e)) {
+      manager.toggleSelection(key);
+    } else {
+      manager.select(key, e);
+    }
```

Comment:
In the build prior to the master merge and in my local that has the master merge + some rough hacks to get selection working again, I can't seem to toggle selection in the ListView via Enter on an already toggled item. Hitting enter without the option+cmd key will enter the `manager.select` case while hitting enter with the option+cmd key works when toggling selection ON but not off (doesn't even seem to trigger onSelect).

Hitting Enter then Enter + Option + CMD is the only way I can get it to untoggle properly